### PR TITLE
Updating Contributing to OptaPlanner section in README

### PR DESCRIPTION
**Pull request by dupliaka**
Change of contribution section + add some related badges + replace link to an OptaPlanner image

Contribution section is a quickest way how to commit changes to OptaPlanner for community.
@emmurphy1 Could I ask you to check it as well, please?

### JIRA

https://issues.redhat.com/browse/PLANNER-2684

### Checklist
- [x] Documentation updated if applicable.
- [ ] Upgrade recipe provided if applicable.

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>specific pull request build</b> please add comment: <b>Jenkins (re)run [optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] tests</b>
* for a <b>full downstream build</b> 
  * for <b>jenkins</b> job: please add comment: <b>Jenkins run fdb</b>
  * for <b>github actions</b> job: add the label `run_fdb`
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
* for a <b>Quarkus LTS check</b> please add comment: <b>Jenkins run LTS</b>
* for a <b>specific Quarkus LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] LTS</b>
* for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
* for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] native</b>
</details>

### CI Status

 You can check OptaPlanner repositories CI status from [Chain Status webpage](https://kiegroup.github.io/optaplanner/).

### CODE REVIEW

File: README.adoc
Comment (yurloc) [inline README.adoc:59]: External contributors do not have the permissions to set reviewers (if we're talking about PRs). Neither are they expected to know who the committers are. Opening a PR is enough. We'll notice.

Comment (dupliaka) [reply README.adoc]: Thanks for that notice

File: README.adoc
Comment (rsynek) [inline README.adoc:59]: I don't think that the link for the CI source code is necessary; hardly any external contributor is interested in that. Let's keep the amount of information in this readme minimal.

File: README.adoc
Comment (rsynek) [inline README.adoc:60]: Let's not mention such details; in the end, it's the reviewer's responsibility to make sure the PR adheres to our standards.

**Review by rsynek — COMMENTED** [review]
Thanks for looking into this! I left there a couple of suggestions to make it even better.

File: README.adoc
Comment (dupliaka) [reply README.adoc]: Removed

File: README.adoc
Comment (rsynek) [inline README.adoc:56]: I think that "welcome" is an adjective in this sentence:

```suggestion
This is an open-source project, and you are more than welcome to contribute!:heart:
```

File: README.adoc
Comment (rsynek) [inline README.adoc:61]: This may sound slightly better:
```suggestion
If you want to contribute or start an opinionated discussion, join our https://groups.google.com/g/optaplanner-dev[discussion] or send an e-mail directly to optaplanner-dev@googlegroups.com.
```

File: README.adoc
Comment (rsynek) [inline README.adoc:66]: Fixes some typos and reformulates the text a bit:
```suggestion
Every change must go through *Pull Request*. Please follow the <<Contributing>> section for more details. OptaPlanner has a CI in place that runs checks against your branch after you submit the PR. Don't worry if something gets wrong or breaks other parts of functionality; our friendly PR maintainers will help you with that.
```

File: README.adoc
Comment (rsynek) [inline README.adoc:26]: It's been a while since we moved them; it's time to change the wording:
```suggestion
OptaPlanner's quickstarts are located in the https://github.com/kiegroup/optaplanner-quickstarts[optaplanner-quickstarts repository].
```

File: README.adoc
Comment (rsynek) [inline README.adoc:63]:
```suggestion
If you want to submit an issue, please check out the https://issues.redhat.com/projects/PLANNER/issues[OptaPlanner Jira project].
```

File: README.adoc
Comment (rsynek) [inline README.adoc:91]: I just tried locally and it took 01:10 min. Let's put there 2 minutes.
```suggestion
- :rocket: *build-fast*: `mvn clean install -Dquickly` skips any tests and code analysis (~2 min)
```

File: README.adoc
Comment (ge0ffrey) [inline README.adoc:88]: The build doesn't fail if the code style is incorrect. Instead, it is automatically reformatted on every build.
Now, reading further, this indeed explained.

But I am wondering if we can TLDR this more, something like:

=== Code standards
Your code is automatically formatted according to the _Import and Code Style_ conventions during every Maven build. CI checks enforces those conventions too, so be sure to build your project with maven before PRing your changes:

... (quickly build command line instruction) ...

... (IDE configuration) ...

File: README.adoc
Comment (ge0ffrey) [inline README.adoc:68]: Code style of adoc files: Use = instead of # for titles
Asciidoctor supports # too, for MarkDown compatibility, but we always use =

```suggestion
=== Contributing
```

File: README.adoc
Comment (ge0ffrey) [reply README.adoc:56]: Can we add a space before the :heart: ?

**Comment by ge0ffrey** [comment]
@dupliaka Can we also move the "Starter issues" section inside (and probably to the bottom) of the contributor section?

File: README.adoc
Comment (dupliaka) [reply README.adoc]: Inlined

File: README.adoc
Comment (dupliaka) [reply README.adoc:56]: Sure, I was planning to invite @emmurphy1 as well

File: README.adoc
Comment (dupliaka) [reply README.adoc:90]: Have you cleaned up your .m2?

File: README.adoc
Comment (dupliaka) [reply README.adoc:67]: FIxed

File: README.adoc
Comment (rsynek) [reply README.adoc:90]: Good point, that might change the build time dramatically.

File: README.adoc
Comment (dupliaka) [reply README.adoc:87]: :+1:

**Comment by dupliaka** [comment]
> @dupliaka Can we also move the "Starter issues" section inside (and probably to the bottom) of the contributor section?

:facepalm: I was absolutely sure that starter issues is a short FAQ session about what happens if you have some issue with build. I suggest to move it to Getting into section

File: README.adoc
Comment (dupliaka) [reply README.adoc:90]: Yup you are right, the average is average)

File: README.adoc
Comment (emmurphy1) [inline README.adoc:69]:
```suggestion
. Create your feature branch. The following example shows how to create and check out the `feature/fooFeature' branch:
 `git checkout -b feature/fooFeature'
```

File: README.adoc
Comment (emmurphy1) [inline README.adoc:70]:
```suggestion
. Commit your changes and add a comment about the changes:
 `git commit -am 'Added a fooFeature'`
```

File: README.adoc
Comment (emmurphy1) [inline README.adoc:71]:
```suggestion
. Push to the branch to GitHub:
 `git push origin feature/fooFeature`
```

File: README.adoc
Comment (emmurphy1) [inline README.adoc:72]:
```suggestion
. Create a new pull request.
```

File: README.adoc
Comment (emmurphy1) [inline README.adoc:76]:
```suggestion
Your code is automatically formatted according to the _Import and Code Style_ conventions during every Maven build. CI checks enforce those conventions too, so be sure to build your project with maven before creating your PR:
```

File: README.adoc
Comment (emmurphy1) [inline README.adoc:82]:
```suggestion
=== Building your OptaPlanner project

Use one of the following ways to build your OptaPlanner project:
```

File: README.adoc
Comment (emmurphy1) [reply README.adoc:91]: Note that I have corrected all headings to sentence case. You can ignore this if the rest of the doc uses title case.

File: README.adoc
Comment (emmurphy1) [inline README.adoc:93]:
```suggestion
You can check the CI status of the OptaPlanner repositories from the https://kiegroup.github.io/optaplanner/[Chain Status webpage].
```

File: README.adoc
Comment (yurloc) [inline README.adoc:84]: The list is broken (see https://github.com/dupliaka/optaplanner/blob/PLANNER-2684/README.adoc#building-your-optaplanner-project). Probably a blank line is needed before the first list item.

Comment (dupliaka) [reply README.adoc:85]: Thanks for noticing! Fixed that, rebased and ready to merge

File: README.adoc
Comment (yurloc) [reply README.adoc:71]: @dupliaka +1 to this.

* No need to capitalize pull request. It's part of the GitHub jargon (branch, fork, workflow, ...) and not a proper name (like GitHub, OptaPlanner, Maven, etc.).
* This list items are full sentences so they should end with a period.

File: README.adoc
Comment (yurloc) [inline README.adoc:79]:
```suggestion
For information about how to set up code style checks, see https://github.com/kiegroup/optaplanner/blob/main/build/optaplanner-ide-config/ide-configuration.adoc[IDE Setup Instructions].
```

**Review by yurloc — COMMENTED** [review]
A few minor issues, otherwise LGTM.

**Comment by dupliaka** [comment]
@yurloc "This list items are full sentences so they should end with a period." 
Thanks for pointing that out, I was using some base knowledge back from university. 
There is indeed different kind of lists, but that I m afraid not the full sentenced list, but chronologically followed steps.
So I edited it a bit according to this article https://www.purchase.edu/editorial-style-guide/general-style-preferences/punctuation/bulleted-and-numbered-lists/

**Review by yurloc — APPROVED** [review]

File: README.adoc
Comment (dupliaka) [reply README.adoc]: Will not change that

File: README.adoc
Comment (dupliaka) [reply README.adoc]: Fixed

**Comment by dupliaka** [comment]
@emmurphy1 and @yurloc  thank you very much for helping with the read me! I merged most of the changes you proposed. Please left your comments till the end of the day so to change it before we merge it! Thank you!

File: README.adoc
Comment (yurloc) [inline README.adoc:41]: Not sure where this is coming from but it breaks the following list formatting.

File: README.adoc
Comment (yurloc) [inline README.adoc:64]:
```suggestion
Every change must be submitted through a GitHub pull request (PR). OptaPlanner uses continuous integration (CI). The OptaPlanner CI runs checks against your branch after you submit the PR to ensure that your PR doesn't introduce errors. If the CI identifies a potential problem, our friendly PR maintainers will help you resolve it.
```

**Pull request merged**
